src/fireflyClient.ts

```diff
@@ -350,8 +342,8 @@ export class FireflyClient {
    * @returns margin balance of positions of given symbol
    */
   async getAccountPositionBalance(symbol: MarketSymbol, perpContract?: address) {
-    const perpV1Contract = this.getContract("PerpetualProxy", perpContract, symbol);
-    const marginBalance = await perpV1Contract.connect(this.getWallet()).getAccountPositionBalance(this.getPublicAddress());
+    const perpV1Contract = this.getContract(this._perpetual, perpContract, symbol);
+    const marginBalance = await perpV1Contract.connect(this.getWallet()).getAccountBalance(this.getPublicAddress());
```

Comment:
the variable name `marginBalance` is misleading as it implies you are getting balance of user in margin bank, but you actually are getting user's position balance, best to remove this variable completely and just write:

`return await perpV1Contract.connect(this.getWallet()).getAccountBalance(this.getPublicAddress());`